Google.Benchmark.nuspec

```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+    <metadata>
+        <id>Google.Benchmark</id>
+        <version>1.0.0</version>
+        <authors>https://github.com/google/benchmark/blob/master/AUTHORS</authors>

```

Comment:
Yes true. I will try to deduce the version number from the latest git tag. Would this work out?